### ISSUE TITLE
Missing attribute `team_id` for MattermostAPIv4

### ISSUE THREAD

**Issue by tferreira** [attzonko/mmpy_bot#36]
Hi, using MattermostAPIv4, I have an issue when using the info plugin as the `get_team_id()` method defined in the dispatcher is trying to use an undefined attribute `team_id`:

https://github.com/attzonko/mmpy_bot/blob/e27363657470c90eb513b9c496d287f153b14168/mmpy_bot/dispatcher.py#L212

Stacktrace:

```
Traceback (most recent call last):
  File "/home/user/src/mmpy_bot/dispatcher.py", line 89, in dispatch_msg
    func(Message(self._client, msg, self._pool), *args)
  File "/home/user/src/mmpy_bot/plugins/info.py", line 9, in info_request
    message.send('TEAM-ID: `%s`' % message.get_team_id())
  File "/home/user/src/mmpy_bot/dispatcher.py", line 212, in get_team_id
    return self._client.api.team_id
AttributeError: 'MattermostAPIv4' object has no attribute 'team_id'
```

Have I missed something or is there something to fix ?

Thank you.

**Comment by seLain** [comment]
Thanks for reporting, it seems a bug. Check out the fix at PR #37 .

**Comment by tferreira** [comment]
Working perfectly, thanks for the quick fix ! \o/
